src/components/new-request/name-input.vue

```diff
@@ -129,10 +129,9 @@ export default class NameInput extends Vue {
     this.setClearErrors(null)
   }
 
-  /* the leading and trailing spaces need to be removed when name input finished
-      Using the sanitizeName, which has been used in name-capture to do the work
-  */
```

Comment:
Would it be better to keep this as JSDoc comment?

Reply:
This comment related to what the code inside the method does, not to the method itself, which is why I moved it.

(Note that old code wasn't in JSDoc format.)

Reply:
Yup, it was missing an asterisk I believe. OK Sev.